### PR TITLE
Ability to bulk fetch and publish all api versions associated with an api

### PR DESCRIPTION
**Pull request by iamnishantgarg**
Added fucntionality to bulk fetch and publish all the versions associated with a particular api

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -347,6 +347,106 @@ async function activate(context) {
 		treeProvider.refresh();
 	});
 
+	vscode.commands.registerCommand('postmanApis.syncApiToPostman', (element) => {
+		let apiVersions = JSON.parse(context.workspaceState.get(element.id));
+		vscode.window.showWarningMessage(`Publish all the versions of the API \`${element.name}\`  to Postman?`, "Yes", "Cancel").then((choice) => {
+
+			if (choice === "Yes") {
+				let disposer = utils.setStatusBarMessage('Uploading the API to Postman');
+				// addind this counter to maintain the count of api versions that has been processed
```
Comment (vinitshahdeo) [inline extension.js]: @iamnishantgarg Typo - adding not adding

**Review by vinitshahdeo — COMMENTED** [review]
@iamnishantgarg I'm not sure why diff looks weird here, is the lint off?

**Comment by iamnishantgarg** [comment]
> @iamnishantgarg I'm not sure why diff looks weird here, is the lint off?

I think lint is fine but we have a tab space of 4 (used for indenting) generally we use 2 may be that’s the reason it looks weird

**Review by vinitshahdeo — COMMENTED** [review]
@iamnishantgarg Isn't your changes should be merged after Srihari's changes? Are they indepedent?

**Pull request merged**